lib/core/form/components/widgets/text/text.widget.spec.ts

```diff
@@ -150,6 +150,14 @@ describe('TextWidgetComponent', () => {
                 expect(inputElement.placeholder).toBe('simple palceholder');
             });
 
+            it('should show the field placeholder when clicked', () => {
```

Comment:
Can you use the async and whenStable method, please?

Reply:
The test case is only to check the value of placeholder when clicked. There is no asynchronous call made. So I didnt add async and whenStable methods.